content/03.methods.md

```diff
@@ -138,18 +138,19 @@ For both these tools we used aligned BAM and chimeric SAM files from STAR as inp
 We ran STAR-Fusion with default parameters and annotated all fusion calls with GRCh38_v27_CTAT_lib_Feb092018.plug-n-play.tar.gz provided in the STAR-fusion release. 
 For Arriba, we used a blacklist file (blacklist_hg38_GRCh38_2018-11-04.tsv.gz) from the Arriba release tarballs to remove recurrent fusion artifacts and transcripts present in healthy tissue.
 We also provided Arriba with strandedness information or set it to auto-detection for polyA samples.
-The RNA expression and fusion workflows can be found in the [KidsFirst GitHub repository](https://github.com/kids-first/kf-rnaseq-workflow).
+We used [FusionAnnotator](https://github.com/FusionAnnotator/FusionAnnotator) to harmonize Arriba fusion annotations with those of STAR-Fusion.
```

Comment:
Can you add some more context here? I know one of these tools automatically does this for you, but I don't remember which one.

Reply:
done